superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx

```diff
@@ -165,6 +173,36 @@ const displayField = ({
     helpText="Pick a nickname for this database to display as in Superset."
   />
 );
+const forceSSLField = ({
+  isEditMode,
+  changeMethods,
+  db,
+  sslForced,
+}: FieldPropTypes) => (
+  <div css={(theme: SupersetTheme) => infoTooltip(theme)}>
+    <Tooltip title={t('SSL is required for Preset')}>
+      <Switch
+        disabled={sslForced && !isEditMode}
+        checked={db?.parameters?.encryption || sslForced}
+        onChange={changed => {
+          changeMethods.onParametersChange({
+            target: {
+              type: 'toggle',
+              name: 'encryption',
+              checked: true,
+              value: changed,
+            },
+          });
+        }}
+      />
+      <span css={toggleStyle}>SSL</span>
+    </Tooltip>
+    <InfoTooltip
+      tooltip={t('SSL mode "require" will be used')}
```

Comment:
"require" is psycopg2 specific

```suggestion
      tooltip={t('SSL will be enabled in the database connection')}
```

Reply:
the figma currently says require. I like this wording better though.

Reply:
Yeah, it doesn't make sense for other drivers. With psycopg2 SSL is enabled with `sslmode='require'`, but other driver have different approaches.

Reply:
yeah I agree, I ran it by Sophie and she agreed.